pkg/tanka/workflow.go

```diff
@@ -185,8 +185,5 @@ func Show(baseDir string, opts Opts) (manifest.List, error) {
 // Eval returns the raw evaluated Jsonnet output (without any transformations)
 func Eval(dir string, opts Opts) (raw interface{}, err error) {
 	r, _, err := eval(dir, opts.JsonnetOpts)
-	if err != nil {
-		return nil, err
-	}
-	return r, nil
+	return r, err
 }
```

Comment:
What's the reason for removing this? Well behaved functions should not return data in case of errors

Reply:
If `r` is not nil, then the evaluation succeeded, the error could still be that there is no Environment but still evaluate the jsonnet fine. Do you think I should catch that err type and only `return r, nil` if it isn't about the missing environment?

Reply:
PTAL, I catch a few errors that have no effect on the eval itself and print them as log messages instead.